app/helpers/katello/subscription_mailer_helper.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  module SubscriptionMailerHelper
+    include Rails.application.routes.url_helpers
+
+    def report_url
+      params = { format: 'html', template_id: ReportTemplate.find_by(name: "Subscription - Entitlement Report").id , input_values: { TemplateInput.find_by_name("Days from Now").id.to_s  => { value: UserMailNotification.first.mail_query.to_s } } }
```

Comment:
I think CSV might be a more suitable format for the report since it's both human and machine readable.

Reply:
Fyi, I asked for feedback about this on Aha. Will pass on the response here